recipe/meta.yaml

```diff
@@ -1,32 +1,34 @@
-{% set version = "0.8.0" %}
+{% set name = "hvplot" %}
+{% set version = "0.8.1" %}
 
 package:
-  name: hvplot
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/h/hvplot/hvplot-{{ version }}.tar.gz
-  sha256: 8630dba34969b105e267cbe14237e2d56d9969c801980ae5e5190bc18b2792d6
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hvplot-{{ version }}.tar.gz
+  sha256: 53ddfe06743c4d5924fb1e597655bfa14e80feaf57a438bfd28442d36ac7c2b6
 
 build:
-  skip: True # [py<36]
   number: 0
+  # panel >=0.13.0 and nodejs aren't available on s390x
+  skip: True # [py<36 or s390x]
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
-    - pyct >=0.4.4
-    - param >=1.12.0
```

Comment:
The host pinning should still match what is set here:
https://github.com/holoviz/hvplot/blob/v0.8.1/pyproject.toml

The point brought up yesterday does not apply here as neither pyct and param are run dependencies, and neither are set in conda_build_config.yaml.
Besides, removing the lower bound without setting an explicit version is equivalent to setting >=0.

Reply:
Thanks, I've updated it. 
Our docs about using pinnings in `host` don't cover all use cases and I'm still a bit confused

Reply:
I agree, pinning right is too complicated at the moment.